framework/components/ecl-links/_links-mixins.scss

```diff
@@ -1,6 +1,7 @@
 @mixin link() {
   color: $ecl-color-primary;
   line-height: 1.66;
```

Comment:
Why? Whenever you place a link in a paragraph, you can break the line-height of it...